qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -214,6 +255,16 @@ def prepare_traces():
     inputs = prepare_data(httpclient)
     triton_client_http.infer("ensemble_add_sub_int32_int32_int32", inputs)
 
+    send_bls_request(model_name="ensemble_add_sub_int32_int32_int32")
+
+
+def send_bls_request(model_name="simple"):
+    with httpclient.InferenceServerClient("localhost:8000") as client:
+        inputs = prepare_data(httpclient)
+        inputs.append(httpclient.InferInput("MODEL_NAME", [1], "BYTES"))
+        inputs[2].set_data_from_numpy(np.array([model_name], dtype=np.object_))
```

Comment:
Use index `-1` rather than `2` if last input to generalize better if prepare_data adds other inputs in future
```suggestion
        inputs[-1].set_data_from_numpy(np.array([model_name], dtype=np.object_))
```